press/index.html

```diff
@@ -33,9 +33,9 @@ <h4>About NewPipe Extractor</h4>
             <hr class="separator" />
 
             <h4>About Team NewPipe</h4>
-            <p>Team NewPipe are the main developers of the NewPipe app and the NewPipe Extractor. At December 2017 they counted eight developers who work voluntarily on this Open Source project. All work of NewPipe's developers is independent and is not funded by any company or governmental institution but by single donations of private persons.</p>
+            <p>Team NewPipe are the main developers of the NewPipe app and the NewPipe Extractor. At April 2020 they counted twelve developers who work voluntarily on this Open Source project. All work of NewPipe's developers is independent and is not funded by any company or governmental institution but by single donations of private persons.</p>
```

Comment:
How is that number composed?

Reply:
People in TNP who set their appartenance in TNP to public. Go to https://github.com/orgs/TeamNewPipe/people in private mode or disconnect, you'll see 12.